core/core.go

```diff
@@ -302,12 +301,6 @@ func (n *IpfsNode) OnlineMode() bool {
 	}
 }
 
-// TODO expose way to resolve path name
-
-func (n *IpfsNode) Resolve(k util.Key) (*merkledag.Node, error) {
-	return (&path.Resolver{n.DAG}).ResolvePath(k.String())
-}
-
```

Comment:
I'm surprised there weren't any call sites. Since we're moving to the "category"-esque world of free functions, would this fit into one of the existing packages?

(I'm in agreement that it should be possible to resolve a path without exposing the path resolver)